tests/cypress/integration/2-general-onboarding-flow/get-started-experience.cy.js

```diff
@@ -65,6 +65,45 @@ describe( 'Start Setup WP Experience Page', function () {
 		} );
 	} );
 
+	const EventsAPI = ( experience_val ) => {
+		cy.intercept('http://localhost:8882/index.php?rest_route=%2Fnewfold-onboarding%2Fv1%2Fevents%2Fbatch&flow=wp-setup&_locale=user').as('events');
+		cy.wait('@events').then((interception) => {
+			const responseBody = interception.request.body;
+			const responseData1 = responseBody[0].data;
+			const responseData2 = responseBody[1].data;
```

Comment:
What is the difference between `responseData1` and `responseData2`? Do we require both of them?

Reply:
This is needed for the edge case, sometimes the experience _level comes into the second data and sometimes in the first. I've refined it more now so that, it doesn't fail. 90% of the time it'll come in the resposeData1.